packages/admin/admin/src/form/FinalFormSearchTextField.tsx

```diff
@@ -25,7 +23,7 @@ export function FinalFormSearchTextField(inProps: FinalFormSearchTextFieldProps
         icon = <Search />,
         placeholder,
         endAdornment,
-        clearable,
+        required,
```

Comment:
Since `endAdornment` and `required` are no longer used, you can simply remove them here, so they're automatically included and applied to `FinalFormInput` from `restProps`. 

```suggestion
```